2018/index.dd

```diff
@@ -94,6 +95,15 @@ $(T table,,
     )
 )
 
+$(T h2,,Summary trailer from DConf16)
+
+<div style="margin-top: -20px;">
+$(P For the ones that missed it (and the ones that didn't too), here is a short video about the conference:
```

Comment:
When you're already writing raw html, I think it's better to not confuse things by intermixing ddoc macros. Not sure, but I think the paragraph should be closed before `iframe` and not after the closing `</div>` tag.

Reply:
Ok. Thanks.